### PR TITLE
speedtest-go: 1.70 -> 1.7.5

### PR DESCRIPTION
**Pull request by luftmensch-luftmensch**
## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by Aleksanaa** [comment]
Thanks for the update. But what formatter are you using? That seems like Alejandra or something.

File: pkgs/tools/networking/speedtest-go/default.nix [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
   src = fetchFromGitHub {
     owner = "showwin";
     repo = pname;
     rev = "refs/tags/v${version}";
-    hash = "sha256-G/ovJk/4pDSDlOUSP9/d0vzTg5IaMPMMPhAN+VGxi/c=";
+    hash = "sha256-8YN7k7sP0S3MIArUfwDYMQvcCKNTxauQjevyOoisAuc=";
   };
 
   vendorHash = "sha256-wQqAX7YuxxTiMWmV9LRoXunGMMzs12UyHbf4VvbQF1E=";
```
Comment (Aleksanaa) [inline pkgs/tools/networking/speedtest-go/default.nix:17]: Are you sure there's no vendorHash update?

Comment (luftmensch-luftmensch) [reply pkgs/tools/networking/speedtest-go/default.nix:17]: It didn't update it. I double checked changing the vendorHash using the `lib.fakeHash`.

**Comment by luftmensch-luftmensch** [comment]
Yeah I'm using alejandra as formatter in the editor, but usually I run `nixfmt-rfc-style` to format the document before commit (I forgot to do it).

File: pkgs/tools/networking/speedtest-go/default.nix (hunk 1 above)
Comment (luftmensch-luftmensch) [reply pkgs/tools/networking/speedtest-go/default.nix:17]: Checking the repository the owner didn't update for about a year.

To double check a possibile change i updated the vendorHash value using the `lib.fakeHash` to see what was the real value.

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**